Additional NR Addon Cars

Zacam opened the pull request:
Post ModLoader Conversion, additional conversions from "replacement" to "addon" occurred after the Polestar.

metonator commented on GameLauncher/App/Program.cs:
```diff
@@ -32,7 +32,7 @@ static class Program {
                 WebClientWithTimeout client = new WebClientWithTimeout();
                 File.Delete("Languages\\Default.lng");
-                client.DownloadFile("https://raw.githubusercontent.com/metonator/GameLauncher_NFSW-translations/master/Languages/English.lng", "Languages\\Default.lng");
+                client.DownloadFile("https://raw.githubusercontent.com/SoapboxRaceWorld/GameLauncher_NFSW-translations/master/Languages/English.lng", "Languages\\Default.lng");
```
This is still under my repository, will return 404 not found

GitMensch replied:
Yes, correct. Before I fix this: what do you think about auto-download all languages on first start?

metonator replied:
Already did in next version, but still causes some errors, so actually
using that silly "onefile" download.

metonator commented:
Already did in next version, but still causes some errors, so actually
using that silly "onefile" download.

W dniu środa, 31 stycznia 2018 Simon Sobisch <notifications@github.com>
napisał(a):

> *@GitMensch* commented on this pull request.
> ------------------------------
>
> In GameLauncher/App/Program.cs
> <https://github.com/SoapboxRaceWorld/GameLauncher_NFSW/pull/35#discussion_r165013825>
> :
>
> > @@ -32,7 +32,7 @@ static class Program {
>              try {
>                  WebClientWithTimeout client = new WebClientWithTimeout();
>                  File.Delete("Languages\\Default.lng");
> -                client.DownloadFile("https://raw.githubusercontent.com/metonator/GameLauncher_NFSW-translations/master/Languages/English.lng", "Languages\\Default.lng");
> +                client.DownloadFile("https://raw.githubusercontent.com/SoapboxRaceWorld/GameLauncher_NFSW-translations/master/Languages/English.lng", "Languages\\Default.lng");
>
> Yes, correct. Before I fix this: what do you think about auto-download all
> languages on first start?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/SoapboxRaceWorld/GameLauncher_NFSW/pull/35#discussion_r165013825>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AF79UVrMIbOgXYk4hjSfcwJMzXRP5060ks5tQENJgaJpZM4RzGKK>
> .
>

GitMensch commented:
I've removed the translation path, the PR should be able to be merged now.

metonator commented:
Merged

Zacam committed:
DiscordRPC JSON Updates

Updated Tracks listing for some name changes and additional tracks, cleaned out unused Event ID's.
Re-ordered and corrected Cars list against gamefiles folder and in-game display matching.

Zacam committed:
Update cars.json

Add NR Polestar 1

Zacam committed:
Update cars.json

Game Localization correction

Zacam committed:
Merge remote-tracking branch 'upstream/interface_v3' into interface_v3

Zacam committed:
Update cars.json

Correct a "Player" police car using virtual reference instead of foldername reference.

Zacam committed:
Merge remote-tracking branch 'upstream/interface_v3' into interface_v3

Zacam committed:
Update cars.json

Add WEv2 Addon Cars

Zacam committed:
Update cars.json

Missed "Lamborghini" for Huracán entry.

Zacam committed:
Merge remote-tracking branch 'upstream/interface_v3' into interface_v3

Zacam committed:
Update cars.json

Missed additional Addon cars from NR

Zacam committed:
Update cars.json

For WEv2:
Added Ferrari F40;
Added Mercedes-Benz 190E 2.5-16 Evolution II;

Zacam committed:
Update cars.json

Add Pagani Huayra

Zacam committed:
Update MainScreen.Designer.cs

Tag "RememberMe" as just that, since it will/does store pw (as hash) along with email address

Zacam committed:
Update cars.json

Break-fix: ' != ,

Zacam committed:
Update MainScreen.cs

"The Server was" reads better than "Server were" while still being unique compared to  -6 ExitCode

Zacam committed:
Update MainScreen.Designer.cs

Zacam committed:
Merge remote-tracking branch 'upstream/interface_v3' into interface_v3

Zacam committed:
CDN List Order

ServerListUpdater: list is no longer in reverse order

Zacam committed:
Revert/Clean RPC Files

Reverted customizations from CARS and EVENTS json files in favor for per-server "append" functions via /MODS/%serverID% lists so that each server can control the release of new data.

Zacam committed:
Followup Clean RPC

Missed a server specific Event

Zacam committed:
Update cars.json

"Stingray" not "Stringray"

Zacam committed:
Update MainScreen.cs

Update exitcode 1 from:
"You just killed nfsw.exe via Task Manager
to:
"The nfsw.exe was exited via Task Manager"
As the "player" isn't always the one doing the exiting

Zacam committed:
Merge remote-tracking branch 'upstream/interface_v3' into interface_v3
